extensions/2.0/Khronos/KHR_interactivity/Specification.adoc

```diff
@@ -1835,6 +1835,48 @@ The graph execution **MAY** be terminated when the amount of pending activations
 
 === State Manipulation Nodes
 
+==== Custom Variable Access
+
+===== Variable Get
+
+[cols="1h,1,2"]
+|===
+| Type | `variable/get` | Get a custom variable value
+| Configuration
+| `int variable` | The custom variable index
+| Output value sockets
+| `T value`  | The custom variable value
+|===
+
+This node gets a custom variable value using the variable index provided by the `variable` configuration value. The type `T` is determined by the referenced variable. The variable index **MUST** be non-negative and less than the total number of custom variables, otherwise the node is invalid.
```

Comment:
Why are variables accessed by index instead of by name? It would make more sense to me to set a variable called `"money"` than one called `27`.